### ISSUE TITLE
Remove ubuntu store from app scope

### ISSUE THREAD

**Issue by ChloeWolfieGirl** [ubports/ubuntu-touch#176]
People using ubuntu 15.04 see a ubuntu store scope at the bottom of the scope, this should be removed due to the scope no longer being supported.

**Comment by NeoTheThird** [comment]
As long as it's still possible to download apps from the store and not all apps are moved yet, i see no harm in leaving it in the images, for 15.04 at least. We should maybe add a notice to use the OpenStore if possible, though.

**Comment by mariogrip** [comment]
I think this should be moved to an smaller icon, and not be so "dominant", maybe move openstore down there?

**Comment by NeoTheThird** [comment]
Nah, i wouldn't move openstore down there. I always found the store icon weird, i never understood, why it had to be in a different shape than the others. Special cases aren't special enough to break the rules. The openstore should just be an app like all the others, imho.

**Comment by ChloeWolfieGirl** [comment]
The ubuntu store is going to disappear in time anyways, so people have to get used to getting apps from openstore rather then the ubuntu store.

The reason the ubuntu store is a bigger icon is because it was a scope rather then an app.

**Comment by mihaelmilea** [comment]
Right now Dekko is found only in the Ubuntu Store... Maybe there are other apps not ported yet to the Open Store.

**Comment by ChloeWolfieGirl** [comment]
There are many for sure which is why we shouldnt get rid of the store entirely, but we should promote and put openstore in the frount of peoples minds when theyre thinking aboutsoftware getting software.

**Comment by NeoTheThird** [comment]
@mihaelmilea Don't worry, for now it will stay. We added a note to prepare the users for the removal though, it will go live in the next ota: https://github.com/ubports/unity-scope-click/commit/2b8818cd193b631b1aaaf156db07fe0457cf7abd

**Comment by mihaelmilea** [comment]
Also, when an app icon is long pressed, it still sends you to the old Ubuntu Store to uninstall the app - instead of the new Open Store.

**Comment by ChloeWolfieGirl** [comment]
@mihaelmilea we're planning on new ways to uninstall apps as well.
See [142](https://github.com/ubports/ubports-touch/issues/142)

**Comment by NeoTheThird** [comment]
https://github.com/ubports/unity-scope-click/pull/2

**Comment by NeoTheThird** [comment]
Some things to look out for suggested by Stefano:

- .click apps can be launched from the scope
- .deb apps can be launched from the scope (e.g. webbrowser-app)
- Removable .click apps can actually be removed, and scope deletes the related icon from the dash
- Install an app from OpenStore and the related icon appears on the dash
- Search for an installed app, and repeat the tests above
- Long press on an icon: app title, app icon, and related actions are properly displayed
- Repeat the tests until the scope crashes :stuck_out_tongue:
- In the "Manage scopes" bottom edge, "Apps" scope is listed and the icon is visible

**Comment by rubencarneiro** [comment]
move openstore down, and change the colour to black, " does it comes in black"

**Comment by NeoTheThird** [comment]
> move openstore down, and change the colour to black, " does it comes in black"

What? I don't understand your question. If you're suggesting to move the openstore down where the ubuntu store used to be, i repeat myself in saying that i think that's a bad idea. The open store is not a scope, it works perfectly fine as an app that you can open, close and navigate nomally. Moving the open-store down there is an inconsistency in the user-experience without any upside. I personally really like how clean the apps scope looks without the ubuntu store...

**Comment by rubencarneiro** [comment]
Create a poll in the website, and let users decide what looks better for them, decisions should nota be restricted to a 3 or 4 developers.
Let users not developers decide.

**Comment by ChloeWolfieGirl** [comment]
If it's replacing where the Ubuntu store is, it should be a scope.

Scopes and apps should be treated differently.

**Comment by NeoTheThird** [comment]
> Create a poll in the website, and let users decide what looks better for them

It's a little short-notice now, since we're releasing in four days...

**Comment by mihaelmilea** [comment]
I'm so glad for the appstore the way it is now with open store - just an app! It was a pain before with Ubuntu Store that it was not possible just to close it - you had to go back, back, back and back in order to be able to use your phone again!

**Comment by sverzegnassi** [comment]
*I'm talking here as OpenStore maintainer and developer.*

The fact that "Ubuntu Store" was a scope does not describe the intimate reason why that "orange box" was there. The fact is that "Ubuntu Store" and "Apps" were actually a single thing, provided by a single package (`unity-scope-click`), with mutual dependencies on each other.

I'd like to point out that users can uninstall the OpenStore client from their devices anytime - if they don't trust what we're doing or they decide they don't need such service.
Even if it comes is pre-installed on Ubuntu devices, UBports hasn't set up any restriction or special priviledge: OpenStore is just a regular apps like any other.

Hardcoding that button in the scope will surely interfere with such ability.

**Comment by Flohack74** [comment]
@ChloeWolfieGirl

**Comment by Flohack74** [comment]
@ChloeWolfieGirl @rubencarneiro  also Openstore Team is not UBports, and its their decision. If you want a vote about this, you need to raise it with the Openstore Team. We stay clear of demanding anything like this full integration since we are already happy that they spend their time and effort to run the Openstore. They decided that a normal App would be the better usecase, and frankly I support this, too. By doing it in this way for example you can update the store anytime by itself, the old integration method would need an OTA release.

**Comment by Flohack74** [comment]
As something is fishy with Openstore not able to update Apps (401 Unauthorized or similar) this becomes a blocker.

**Comment by sverzegnassi** [comment]
Btw, in which terms would the 401 error be related to the Ubuntu Store removal?

I would rather bet on some bad implementation of OpenStore APIs into system-settings app, or on some temporary problem with the servers. I feel like the 'critical' label here is misplaced.

**Comment by Flohack74** [comment]
I updated now clock and telegram via Openstore, seems to work. So WFM, maybe this was smth else ;) - If nothing appears, will remove critical label tomorrow...

**Comment by NeoTheThird** [comment]
#184 is the issue, this one has most likely nothing to do with that.

**Comment by ChloeWolfieGirl** [comment]
I wasn't saying we should demand Openstore to make a scope, but rather, that would have to happen before we even considered moving it to the bottom, and as a rectangle, not that we should force them to make a scope.

When using the Ubuntu store scope you didn't have to press the back buttons, but slide your finger all the way from the left to right, or press the Ubuntu logo to bring you back to the first scope, so I don't fully agree that it was more annoying then an app, I think the store as a scope and store as an app, play different means to the same ends, and arguably are aimed at slightly different markets.

**Comment by ivoxavier** [comment]
Hi everyone, if we swipe up from the app scope we get its subpage, there at the right top corner an shortcut to ubuntu store still exists. When pressed it takes us back to app scope. 
![screenshot20171219_221727358](https://user-images.githubusercontent.com/16002474/34181792-8d3ffb30-e50b-11e7-8969-f72e88f60f9e.png)

**Comment by NeoTheThird** [comment]
Right, keen observation @ivoxavier! @sverzegnassi left this one in, because it's hardcoded in unity8 itself, and not part of the unity-click-scope package. We did not want to touch unity8 for this release, so we decided to leave that button in. But good catch!

**Comment by andybleaden** [comment]
And also that button just shows my apps not the Ubuntu Store @ivoxavier!
